ckanext/canada/schemas/presets.yaml

```diff
@@ -3117,6 +3125,7 @@ presets:
     - value: JSON
       mimetype: application/json
       openness_score: 3
+    - value: JSON-LD
```

Comment:
looks like the file extension has no hyphen. so `JSONLD`, optionally with a label.